### PR TITLE
adds audio phase shift command #195

### PR DESCRIPTION
**Pull request by ablwr**
#195 

CC @bturkus, whom I cannot tag as a reviewer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1110,6 +1110,30 @@
...
   <h4>Normalize/Equalize Audio</h4>
 
+  <!-- phase shift -->
+  <span data-toggle="modal" data-target="#phase_shift"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Inverses the audio phase of the second channel">Flip phase shift</button></span>
+  <div id="phase_shift" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Flip audio phase shift</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -af pan="stereo| c0=c0| c1=-1*c1" <i>output_file</i></code></p>
```
Comment (retokromer) [inline index.html]: Is there a reason for the spacing? I would suggest
```pan="stereo|c0=c0|c1=-1*c1"```
(preferred) or
```pan="stereo | c0=c0 | c1=-1*c1"```
.

File: index.html [hunk 2]
```diff
@@ -1110,6 +1110,30 @@
...
+          <p>This command inverses the audio phase of the second channel by shifting it 180°.</p>
```
Comment (retokromer) [inline index.html]: Does `shifting` also include a rotation in English? If so, then ignore my comment.

File: index.html [hunk 3]
```diff
@@ -1110,6 +1110,30 @@
...
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-af</dt><dd>specifies that the next section should be interpreted as an audio filter</dd>
+            <dt>pan=<dt><dd>tell the quoted text below to use the <a href="https://ffmpeg.org/ffmpeg-filters.html#pan-1" target="_blank">pan filter</a></dd>
+            <dt>"stereo| c0=c0| c1=-1*c1"</dt><dd>maps the output's first channel (c0) to the input's first channel and the output's second channel (c1) to the inverse of the input's second channel</dd>
```
Comment (retokromer) [inline index.html]: spacing as above

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html]: Just readibility? I cp/pasted from @privatezero in the issue's comments. just updated to the no-spaces which is more standard, I think.

Comment (retokromer) [reply index.html]: Then I would suggest my 2<sup>nd</sup> variant with spaces on both sides of `|`.

File: index.html (hunk 2 above)
Comment (ablwr) [reply index.html]: rotating is probably better

**Review by retokromer — COMMENTED** [review]
Despite my nit-picking, LGTM

**Review by retokromer — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (retokromer) [reply index.html]: Sorry, I was mathematician in my previous life.

**Pull request merged**